### ISSUE TITLE
please support --version

### ISSUE THREAD

**Issue by rubyFeedback** [jirka-h/haveged#42]
Heya,

It would be nice if --version would be supported by the sbin/haveged.

That way I can embed haveged into a commandline script, and display whether the local
computer needs an update to haveged (I tend to compile from source, so it is useful to
me to be able to verify the versions in general).

**Comment by jirka-h** [comment]
Hi,

yes, it makes totally sense to add support for --version. I will add it in the next release.

**Comment by jirka-h** [comment]
Fixed with commit 83d7b228ca36f58779fab1426752dcf6b1ddf447

**Comment by jirka-h** [comment]
Released in 1.9.13

```
$haveged --version
haveged 1.9.13

Copyright (C) 2018-2020 Jirka Hladky <hladky.jiri@gmail.com>
Copyright (C) 2009-2014 Gary Wuertz <gary@issiweb.com>
Copyright (C) 2011-2012 BenEleventh Consulting <manolson@beneleventh.com>

License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```
